textproc/extractpdfmark/Portfile

```diff
@@ -29,6 +29,11 @@ checksums           rmd160  1f1bac62afac8b52e54a796084d5b6c87453122e \
 
 depends_build       port:pkgconfig
 
+# malformed object (unknown load command 2)
+if {${os.platform} eq "darwin" && ${os.major} <= 10} {
+    depends_build-append  port:cctools
+}
```

Comment:
The os version check is unnecessary as the cctools port 'does the right thing' on all platforms. So I would remove it.